doc/using_tf.rst

```diff
@@ -133,6 +133,34 @@ In your training script the channels will be stored in environment variables ``S
    ``output_path``.
 
 
+Use third-party libraries
+-------------------------
+
+If there are other packages you want to use with your script, you can use a ``requirements.txt`` to install other dependencies at runtime.
+
+For training, support for installing packages using ``requirements.txt`` varies by Tensorflow version as follows:
+
+- For TensorFlow 1.11 or newer using Script Mode without Horovod, Tensorflow 1.15.2 with Python 3.7 or newer, and Tensorflow 2.2 or newer:
```

Comment:
Script to script , is there a link you can provide to help users understand about what is script mode?

Reply:
This example notebook is already linked on line 94 of this doc: https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/tensorflow_script_mode_training_and_serving/tensorflow_script_mode_training_and_serving.ipynb